deploy/crds/workspaces.ecd.eclipse.org_devworkspaces_crd.yaml

```diff
@@ -474,32 +478,40 @@ spec:
                       kubernetes:
                         description: Kubernetes component
                         properties:
-                          alias:
-                            type: string
                           inlined:
                             description: Reference to the plugin definition
                             type: string
                           locationType:
                             description: Type of Kubernetes-like location
                             type: string
+                          name:
+                            description: Optional name that allows referencing the
+                              component in commands, or inside a parent
+                            type: string
                           url:
                             description: Location in a plugin registry
                             type: string
+                        required:
+                        - name
                         type: object
                       openshift:
                         description: Openshift component
                         properties:
-                          alias:
-                            type: string
                           inlined:
                             description: Reference to the plugin definition
                             type: string
                           locationType:
                             description: Type of Kubernetes-like location
                             type: string
+                          name:
+                            description: Optional name that allows referencing the
+                              component in commands, or inside a parent
+                            type: string
                           url:
```

Comment:
Same as for k8s: name should be mandatory

Reply:
fixed